external/googleapi/g_utils.py

```diff
@@ -19,9 +19,25 @@ def parse_lat_lng(geo_result_obj):
 
 
 def normalize_us_address(address) -> Optional[Address]:
-    if not address:
+    if not address or address == "":
```

Comment:
As far as I know, an empty string is falsy in python. Do we need `address == ""`?

https://stackoverflow.com/a/9573259

Reply:
Oh you might be right. My brain is stuck in JavaScript mode, I'll remove and retest when I get back home

Reply:
you were correct, I have changed it back.